test/blackbox-tests/test-cases/watching/promotion-db.t

```diff
@@ -0,0 +1,71 @@
+Demonstrate bug where the RPC promotion doesn't print a warning where the normal promotion does.
```

Comment:
It the test "demonstrating a bug"? Or is it testing functionality to prevent regression of a bug?